configure.ac

```diff
@@ -6666,6 +6666,7 @@ if test "x$ac_system" = "xLinux"; then
   plugin_mcelog="yes"
   plugin_mdevents="yes"
   plugin_memory="yes"
+  plugin_mmc="yes"
```

Comment:
building mmc by default will also cause another CI issue (the debian one).

We'll have to look into the way CI is run by using pre-built containers, which adds a form of indirection.